src/main/java/online/talkandtravel/service/impl/ChatServiceImpl.java

```diff
@@ -74,6 +76,14 @@ public class ChatServiceImpl implements ChatService {
   private final UserMapper userMapper;
   private final UserRepository userRepository;
   private final UserChatMapper userChatMapper;
+  private final IAuthenticationFacade authenticationFacade;
+
+  @Override
+  public ChatDto createCountryChat(NewChatRequest request) {
```

Comment:
I think it would be better this method return another dto. Now it is called PrivateChatInfoDto, but maybe it would be renamed to ChatInfoDto and used here. That dto do not contain arrays of messages, but only count of them. Actually in this case it not so necessary, but i feel it might be better, what do you think?

Reply:
this method returns just created entity DTO, and it doesn't have messages and events, therefore, there is no need to count their number

Reply:
ok